internal/service/implement/auth_service.go

```diff
@@ -147,3 +155,27 @@ func (service *AuthService) ValidateRefreshToken(ctx *gin.Context, customerId in
 	}
 	return refreshToken, nil
 }
+
+func (service *AuthService) SendOTPToMail(ctx *gin.Context) error {
+	// generate otp
+	otp := mail.GenerateOTP(6)
+
+	// store otp in redis for 1 minute
+	key := "otp"
```

Comment:
?

Reply:
this is old commit, i already update to include user_id in the key